scania-digital-design-system/tegel
# test(table): testing input wrapper interactions

## **Describe pull-request**  
Created test for table-input-wrapper.

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3448](https://tegel.atlassian.net/browse/CDEP-3448)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to root of the project
2. Make sure you have fetched and checkout `test/CDEP-3448-tables-tests`
3. run `npm run test`

## **Checklist before submission**


[CDEP-3448]: https://tegel.atlassian.net/browse/CDEP-3448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
